app/trip/hooks/useTripsGet.ts

```diff
@@ -5,6 +5,7 @@ import { useTripsCollectionQuery, TripsOrderBy } from '@generated/api'
 export const useTripsGet = (searchWord?: string) => {
   const userId = useUserId()
 
+  // TODO - need to change as currently just showing all trips user created. We also need to show invited trips
```

Comment:
This is the one I asked on Slack.
It's complicated so I created the issue : https://github.com/tabi-memo/frontend/issues/37